File: Histogram - H1/Histogram - H1.dart
String histogram(List<int> results) {
  List<String> result = [];
  for(int i=1;i<=results.length;i++){
    if(results[i-1]!=0){
      result.add("${i}|"+"#"*results[i-1]+" ${results[i-1]}");
    }else{
      result.add("${i}|"+"#"*results[i-1]);
    }
  }
  result  = result.reversed.toList();
  return result.join("\n")+"\n";
}

File: Histogram - H1/Histogram - H1.py
def histogram(results):
    result = []
    for i,v in enumerate(results):
        if v==0:
            result.append(f'{i+1}|{(v)*"#"}')
        else:
            result.append(f'{i+1}|{(v)*"#"} {v}')
    result = reversed(result)
    return "\n".join(result)+"\n"
